packages/core/src/components/stepper/stepper.scss

```diff
@@ -0,0 +1,118 @@
+.atom-stepper {
+  cursor: default;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  list-style: none;
+
+  @include above(medium) {
+    flex-direction: row;
+    height: 100%;
+  }
+}
+
+.number {
+  align-items: center;
+  border: 1px solid var(--color-contextual-success-light-1);
+  border-radius: var(--border-radius-full);
+  color: var(--color-contextual-success-light-1);
+  display: flex;
+  font: var(--text-body-small);
+  height: var(--spacing-large);
+  justify-content: center;
+  letter-spacing: var(--text-body-small-letter);
+  margin-bottom: var(--spacing-base);
+  width: var(--spacing-large);
+}
+
+.step {
+  display: flex;
+  height: var(--spacing-large);
+  list-style-type: none;
+  position: relative;
+  width: 100%;
+
+  &::after {
+    background-color: var(--color-neutral-light-3);
+    content: '';
+    height: 100%;
+    left: var(--spacing-xsmall);
+    position: absolute;
+    top: var(--spacing-xxsmall);
+    width: 1px;
+  }
+
+  &:last-child::after {
+    content: none;
+    display: none;
+  }
+
+  @include above(medium) {
+    align-items: center;
+    flex-direction: column;
+    height: 100%;
+    justify-content: center;
+
+    &:not(:last-child)::after {
+      height: 1px;
+      left: 60%;
+      top: var(--spacing-small);
+      width: 80%;
+    }
+  }
+}
+
+.title {
+  color: var(--color-neutral-regular);
+  font: var(--text-body-small);
+  letter-spacing: var(--text-body-small-letter);
+}
+
+.active {
```

Comment:
We have used the is/has pattern for these cases. Guess it should be be:
 
```CSS
.title {

&.is-active {
// do something
}
}
```

Reply:
Nice!

Made: https://github.com/juntossomosmais/atomium/pull/726/commits/be7d94e5ab5adf29c1017260d4ebd1c73c4c2540